templates/staff/dashboards/index.html

```diff
@@ -26,7 +26,7 @@
 
           {% url "staff:dashboards:repos" as staff_dashboards_repos_url %}
           {% #list_group_rich_item title="Private Repos" type="Dashboard" url=staff_dashboards_repos_url %}
-            <p>Private repos attached to one or more workspaces, where code was first run more than 11 months ago</p>
+            <p>Private research repositories linked to one or more workspaces, with initial code execution over 11 months ago, for IG monitoring</p>
```

Comment:
I'd leave off the "for IG monitoring" for now. I think we don't have a good handle on who uses these dashboards for what purposes and I'm not sure adding this gains us anything.

Reply:
Agreed!